app/views/stats/index.html.erb

```diff
@@ -0,0 +1,87 @@
+<div class='header-content'>
+
+	<p id='notice'><%= notice %></p>
+	
+	<h1>Listing Expenses</h1>
+
+	<div class="container">
+		<div class="bs-example">
+		    <ul class="nav nav-tabs" id="myTab">
+		        <li class="active"><a href="#totalExpensesRevenues">Total of Expenses and Revenues</a></li>
+		        <li><a href="#userExpenses">User Expenses</a></li>
+		        <li><a href="#userRevenues">User Revenues</a></li>
+		        <li><a href="#categorieExpenses">Combination Chart</a></li>
+		        <li><a href="#combination">Category Expenses and Revenues</a></li>
+		    </ul>
+		    <div class="tab-content">
+		        <div id="totalExpensesRevenues" class="tab-pane fade in active">
+		            <div class="container">
+				 		<%= high_chart("my_id0", @chart0) do |c| %>
+				  		<% end %>
+					</div>
+		        </div>
+		        <div id="userExpenses" class="tab-pane fade">
+		        	<div class="container">
+				    	<%= high_chart("my_id1", @chart1) do |c| %>
+				    	<%= raw "options.tooltip.formatter = function() {return '<b> user: '+ this.point.name +'</b>: '+ this.y +' %';}"  %>
+				      	<%= raw "options.plotOptions.pie.dataLabels.formatter = function() { if (this.y > 5) return this.point.name; }"  %>
+				  		<% end %>
+					</div>   
+		        </div>
+		        <div id="userRevenues" class="tab-pane fade">
+		        	<div class="container">
+				    	<%= high_chart("my_id2", @chart2) do |c| %>
+				        <%= raw "options.tooltip.formatter = function() {return '<b> user: '+ this.point.name +'</b>: '+ this.y +' %';}"  %>
+				        <%= raw "options.plotOptions.pie.dataLabels.formatter = function() { if (this.y > 5) return this.point.name; }"  %>
+				  		<% end %>
+					</div>   
+		        </div>
+		        <div id="categorieExpenses" class="tab-pane fade">
+		        	<div class="container">
+				    	<%= high_chart("my_id3", @chart3) do |c| %>
+				  		<% end %>
+					</div>
+		        </div>
+
+		        <div id="combination" class="tab-pane fade">
+		        	<div class="container">
+				    	<%= high_chart("my_id4", @chart4) do |c| %>
+				  		<% end %>
+					</div>
+		        </div>
+			</div>
+	    </div>
+	</div>		
+
+</div>
+
+
+
+
+
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script> 
+<script src="http://code.highcharts.com/highcharts.js"></script>
+<script src="http://code.highcharts.com/modules/exporting.js"></script>
+
+
+
+
+
+
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
```

Comment:
It isn't necessary to put he css/js files here. Put it in the assets/application.css/js